src/controllers/api/reactions.js

```diff
@@ -0,0 +1,49 @@
+const { Thought } = require("../../models");
+
+const createReactionForThought = async (req, res) => {
+  try {
+    //get thought id from req params
+    const { thoughtId } = req.params;
+
+    //push new reaction inside the reactions array for that thought id
+    const newReaction = await Thought.findByIdAndUpdate(
+      thoughtId,
+      {
+        $push: { reactions: { ...req.body } },
```

Comment:
Again, as I commented in your `friends.js` controller, we shouldn't trust that the `req.body` will contain the correct information. We want to protect our database, so please read the comment in that file and apply here too.

Reply:
Done :) 
Thank you!